src/componentprocessor/index.ts

```diff
@@ -114,6 +125,24 @@ async function processXmlTree(
         }
     });
 
+    nodeDefMap.forEach(nodeDef => {
+        let xmlNode = nodeDef.xmlNode;
+        if (xmlNode) {
+            nodeDef.children = getChildren(xmlNode);
+            let parentNode = xmlNode.attr.extends;
```

Comment:
Can we name this something else like _superNode_ or _inheritedNode_ or something to be less ambiguous from the parent/child relationships?

Reply:
Talked with Jorge offline.  We suggested _baseNode_.